lib/api/bucketPutACL.js

```diff
@@ -261,6 +263,9 @@ function bucketPutACL(authInfo, request, log, callback) {
                         return next(null, bucket, revisedAddACLParams);
                     });
             }
+            if (hasError) {
```

Comment:
should be earlier in the code (eright after the `jsonGrants.forEach` loop) : otherwise we may continue while we had an error, and risk calling the callback again (like on line 263)